Can not build v0.11 on Fedora19

/usr/bin/c++   -Dcocaine_core_EXPORTS -O2 -g  -fPIC -I/home/zbr/rpmbuild/BUILD/libcocaine-core2-0.11.2.3/include -I/home/zbr/rpmbuild/BUILD/libcocaine-core2-0.11.2.3/foreign/backward-cpp -I/home/zbr/rpmbuild/BUILD/libcocaine-core2-0.11.2.3/foreign/jsoncpp-0.6.0-rc2/include -I/usr/include/libev    -std=c++0x -W -Wall -Werror -pedantic -o CMakeFiles/cocaine-core.dir/src/drivers/fs.cpp.o -c /home/zbr/rpmbuild/BUILD/libcocaine-core2-0.11.2.3/src/drivers/fs.cpp
In file included from /home/zbr/rpmbuild/BUILD/libcocaine-core2-0.11.2.3/include/cocaine/detail/actor.hpp:24:0,
                 from /home/zbr/rpmbuild/BUILD/libcocaine-core2-0.11.2.3/src/actor.cpp:21:
/home/zbr/rpmbuild/BUILD/libcocaine-core2-0.11.2.3/include/cocaine/common.hpp:52:16: error: operator '||' has no left operand
 #if HAVE_GCC47 || TARGET_OS_MAC
                ^
In file included from /home/zbr/rpmbuild/BUILD/libcocaine-core2-0.11.2.3/include/cocaine/app.hpp:24:0,
                 from /home/zbr/rpmbuild/BUILD/libcocaine-core2-0.11.2.3/src/app.cpp:21:
/home/zbr/rpmbuild/BUILD/libcocaine-core2-0.11.2.3/include/cocaine/common.hpp:52:16: error: operator '||' has no left operand
 #if HAVE_GCC47 || TARGET_OS_MAC
                ^

$ gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/libexec/gcc/x86_64-redhat-linux/4.8.3/lto-wrapper
Target: x86_64-redhat-linux
Configured with: ../configure --prefix=/usr --mandir=/usr/share/man --infodir=/usr/share/info --with-bugurl=http://bugzilla.redhat.com/bugzilla --enable-bootstrap --enable-shared --enable-threads=posix --enable-checking=release --with-system-zlib --enable-__cxa_atexit --disable-libunwind-exceptions --enable-gnu-unique-object --enable-linker-build-id --with-linker-hash-style=gnu --enable-languages=c,c++,objc,obj-c++,java,fortran,ada,go,lto --enable-plugin --enable-initfini-array --enable-java-awt=gtk --disable-dssi --with-java-home=/usr/lib/jvm/java-1.5.0-gcj-1.5.0.0/jre --enable-libgcj-multifile --enable-java-maintainer-mode --with-ecj-jar=/usr/share/java/eclipse-ecj.jar --disable-libjava-multilib --with-isl=/builddir/build/BUILD/gcc-4.8.3-20140624/obj-x86_64-redhat-linux/isl-install --with-cloog=/builddir/build/BUILD/gcc-4.8.3-20140624/obj-x86_64-redhat-linux/cloog-install --with-tune=generic --with-arch_32=i686 --build=x86_64-redhat-linux
Thread model: posix
gcc version 4.8.3 20140624 (Red Hat 4.8.3-1) (GCC) 

$ git branch -v | grep v0.11
- v0.11  399ce63 Merge pull request #124 from diunko/feature/hostname-option

--- a/cocaine-bf.spec
+++ b/cocaine-bf.spec
@@ -2,8 +2,8 @@
 
 Summary:	Cocaine - Core Libraries
 Name:		libcocaine-core2
-Version:	0.11.2.3
-Release:	2%{?dist}
+Version:	0.11.2.8
+Release:	1%{?dist}
 
 
 License:	GPLv2+
@@ -134,6 +134,9 @@ rm -rf %{buildroot}
 %{_sysconfdir}/cocaine/cocaine-default.conf
 
 %changelog
+* Fri Sep 05 2014 Evgeny Safronov <division494@gmail.com> 0.11.2.8-1
+- Bugfix: properly check platform macro definition.
+
 * Wed Jun 11 2014 Evgeniy Polyakov <zbr@ioremap.net> 0.11.2.3-1
 - Updated spec to the latest version to date
 

--- a/debian/changelog
+++ b/debian/changelog
@@ -1,6 +1,12 @@
+cocaine-core (0.11.2.8) unstable; urgency=low
+
+  * Bugfix: properly check platform macro definition.
+
+ -- Evgeny Safronov <division494@gmail.com>  Fri, 05 Sep 2014 07:27:02 +0400
+
 cocaine-core (0.11.2.7) unstable; urgency=low
 
- * Backport locator.hostname config option to v0.11 branch
+  * Backport locator.hostname config option to v0.11 branch
 
  -- Dmitry Unkovsky <diunko@yandex-team.ru>  Fri, 22 Aug 2014 00:38:38 +0400
 

--- a/include/cocaine/common.hpp
+++ b/include/cocaine/common.hpp
@@ -49,7 +49,7 @@
     _name_&                                     \
     operator=(const _name_& other) = delete;
 
-#if HAVE_GCC47 || TARGET_OS_MAC
+#if defined(HAVE_GCC47) || defined(TARGET_OS_MAC)
     #define COCAINE_HAVE_FEATURE_STEADY_CLOCK
     #define COCAINE_HAVE_FEATURE_UNDERLYING_TYPE
 #endif

